app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -154,6 +171,52 @@ public Study createStudy(Study study) {
 
         return study;
     }
+
+    public Study createSynapseProjectTeam(Long userId, Study study) throws SynapseException {
+        // create synapse project and team
+        Team team = new Team();
+        team.setName(randomAlphanumeric(5) + "Bridge-Team");
+        Project project = new Project();
+        project.setName(randomAlphanumeric(5) + "Bridge-Project");
```

Comment:
Use the study's name here, e.g. study.getName() + " Access Team" and study.getName() + " Project"